UI/Web/src/app/manga-reader/_components/manga-reader/manga-reader.component.html

```diff
@@ -194,6 +194,8 @@
                 <option value="full-width">{{t('width')}}</option>
                 <option value="original">{{t('original')}}</option>
               </select>
+              <label for="page-fitting-slider" class="form-label">Manual Width</label>
+              <input id="page-fitting-slider" type="range" min="0" max="100" class="form-range" formControlName="widthSlider">
```

Comment:
I think we shouldn't allow 0. I think range from 1-100 is appropriate.

Reply:
### refering to the other comment as well, you suggested to let 0 be the value to turn it off, so if 0 is inputed the default behaviour of the width mode is restored. That is also why it defaults to 0 instead of 100%